.cirun.yml

```diff
@@ -0,0 +1,17 @@
+# Self-Hosted Github Action Runners on AWS via Cirun.io
+# Reference: https://docs.cirun.io/reference/yaml.html
+runners:
+  - name: arm-runner
+    # Cloud Provider: Amazon Web Services
+    cloud: aws
+    # AWS A1 Arm system
+    instance_type: a1.4xlarge
+    # Ubuntu 20.04, 64 GB drive
+    machine_image: ami-049db9debd560491a
+    region: eu-west-1
+    preemptible: false
+    # Relevant workflow file
+    workflow: .github/workflows/test-cirun.yml
```

Comment:
We recently added a [new feature](https://twitter.com/AktechLabs/status/1551820040408006661) to simply match/create runners by labels instead of workflow file path, which means this param (`workflow`) is no longer required as long as you have `labels`.

Reply:
@aktech awesome, updated!